scripts/generate_provenance

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+# Takes as input a TOML file, and builds a binary and its provenance from the
+# Git HEAD, using the build config in the TOML file. To run locally, pass the
```

Comment:
AFAICT it does not build from HEAD, but from the currently checked out commit? Or maybe that's also called HEAD (perhaps local HEAD)?